python/perspective/perspective/handlers/__init__.py

```diff
@@ -11,8 +11,20 @@
 #  ┗━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┛
 
 from .common import PerspectiveHandlerBase
-from .aiohttp import PerspectiveAIOHTTPHandler
-from .starlette import PerspectiveStarletteHandler
-from .tornado import PerspectiveTornadoHandler
 
-__all__ = ["PerspectiveHandlerBase", "PerspectiveAIOHTTPHandler", "PerspectiveStarletteHandler", "PerspectiveTornadoHandler"]
+try:
+    from .aiohttp import PerspectiveAIOHTTPHandler
+except ImportError:
+    ...
+
+try:
+    from .starlette import PerspectiveStarletteHandler
+except ImportError:
+    ...
+
+try:
+    from .tornado import PerspectiveTornadoHandler
+except ImportError:
+    ...
+
+__all__ = ["PerspectiveHandlerBase", "PerspectiveAIOHTTPHandler", "PerspectiveStarletteHandler", "PerspectiveTornadoHandler" ]
```

Comment:
if the above imports fail, this will cause attribute errors when importing.

Reply:
Ok I was thinking about something like this, should deal with that, this way if it only puts it to __all__ if it passes. What do you think?

try:
    from .aiohttp import PerspectiveAIOHTTPHandler
    __all__.append("PerspectiveAIOHTTPHandler")
except ImportError: